packages/good-design/src/apps/goodid/screens/theme.ts

```diff
@@ -24,15 +24,31 @@ const OnboardScreen = {
   }
 };
 
+const OffersAgreement = {
+  baseStyle: {
+    width: "100%",
+    paddingX: 4,
+    alignItems: "center",
+    styles: {
+      buttonContainer: {
+        space: 4
+      },
+      image: {
+        width: 150,
+        height: 113
+      }
```

Comment:
shouldnt image width be relative to page width?
you need to consult with Sana on these things, think more about how things look in different widths

Reply:
Everywhere on figma the images on the good-id screens are added with a fixed size.
@SanaJamm and me will have a call asap to discuss this all

Reply:
Confirmed this is fixed-sizing